public/assets/ts/admin/editor-loader.ts

```diff
@@ -10,7 +10,7 @@ class EditorLoader {
     const editorsContainer = document.getElementById("editors-container");
 
     for (const source of sources) {
-      new Editor(editorsContainer, source.endpoint, source.response);
+      new window["Editor"](editorsContainer, source.endpoint, source.response);
```

Comment:
Yes! Issue here: #69

It can be done, but ES6 modules take a bit of time to set up and I didn't want to do that as a part of this PR (or the previous ones).  It's a good idea to look into as a task of its own though, for sure.